### DISCUSSION TITLE
Every time i create a acount and hack using wills cheat menu the acount is instantly deleated!!!!!!!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#910] [(B) Bug Discussion]
pls help

**Comment by SakuraKitsunee** [comment]
@BETPRODIGYHACKER i'm pretty sure it's because the hacks are down, but you might want to confirm with PatheticMustan

**Reply by ghost** [reply]
say i creat a acount pass:bob12234 user 2342334234 then i play for 2 hours -not realy as soon as im done and go do something else and close out i use the exact same login info and boooooooom it says 'wrong login info'

**Comment by ghost** [comment]
they are down, it says here
 https://github.com/Prodigy-Hacking/ProdigyMathGameHacking#we-are-aware-hacks-may-be-down-for-some

**Reply by ghost** [reply]
no there not it has allways happened even a month ago

**Reply by ghost** [reply]
allso im talking about wills cheat menu

**Reply by ghost** [reply]
well, @BETPRODIGYHACKER they have probably  been down for a month then

**Reply by ghost** [reply]
oh

**Reply by ghost** [reply]
when will they be fixed?

**Reply by SakuraKitsunee** [reply]
no they were up last night, are you on a school computer?

**Reply by ghost** [reply]
it also happed 2 or even 3 months ago- ive been using wills cheat menu for 3-4 months

**Reply by ghost** [reply]
on chrome

**Reply by SakuraKitsunee** [reply]
talk to PatheticMustan

**Reply by ghost** [reply]
yes right bow but i play on my own puter my school blocks prodigy can u hep meh get around this???

**Reply by ghost** [reply]
*now not bow

**Reply by ghost** [reply]
kay

**Reply by ghost** [reply]
can u help meh unblock prodigy though

**Reply by SakuraKitsunee** [reply]
make a issue and choose the bug one and put what happend, someone will help sooner or later

**Reply by ghost** [reply]
k

**Reply by ghost** [reply]
can someone pls add a obtain all titans button if so i would create a video about u guys thanks
